src/timeline/Timeline.tsx

```diff
@@ -186,12 +208,22 @@ const Timeline = (props: TimelineProps) => {
     });
 
     return (
-      <View>
-        <View style={{marginLeft: HOURS_SIDEBAR_WIDTH}}>{events}</View>
+      <View style={{marginLeft: index === 0 ? HOURS_SIDEBAR_WIDTH : undefined, width: width / numberOfDays}}>
```

Comment:
Have you considered passing `flex: 1` instead of the fixed width. 
It's way more dynamic and probably will work better with orientation change